userspace/libscap/ringbuffer/devset.c

```diff
@@ -66,7 +69,7 @@ void devset_free(struct scap_device_set *devset)
 		{
 			munmap(dev->m_bufinfo, dev->m_bufinfo_size);
```

Comment:
`munmap` is unix only.

Reply:
So here we cannot unmap memory :/

Reply:
yeah but we did not map it in the first place (mmap too is missing; and mmap is used by scap_kmod and scap_bpf that are not built on windows)

Reply:
sounds good, thank you for the explanation :)